packages/experimental/Divider/SPEC.md

```diff
@@ -0,0 +1,5 @@
+# Divider
+
+## Background
+
+Please write here you documentation...
```

Comment:
Followup or expected to be completed with this PR?

Reply:
This will most likely be a follow up.